### DISCUSSION TITLE
Do you use Sanboxie for testing app manifests? If yes, how do you do it?

### DISCUSSION THREAD

**Discussion by Vedabahu** [ScoopInstaller/Extras#9846] [Q&A]
As the title says. Do you use sandboxie?
If yes, how to use it to test out the app manifests that you have created?

**Comment by jasongodev** [comment]
You open Windows Powershell via Sandboxie's run command or start menu.

After you test things you can delete contents of the sandbox and start again.

It's better if you don't have the target manifest in your "unsandboxed" base OS.

**Comment by HUMORCE** [comment]
I use VM instead of Sandbox, because I need to keep Hyper-V off, this PR will be a good answer to your question: https://github.com/ScoopInstaller/Scoop/pull/5349

**Comment by HUMORCE** [comment]
I started testing manifests via Windows Sandbox, now.

Sandbox config and setup scripts:
https://gist.github.com/HUMORCE/062b2ae684ca23a04e802f0401587a92

**Comment by rasa** [comment]
See https://github.com/ScoopInstaller/Scoop/pull/5349

**Comment by Fredolx** [comment]
I would recommend using a VM instead, Sandboxie isn't as safe as you think
